apps/hash-frontend/src/pages/[shortname]/types/entity-type/[...slug-maybe-version].page/definition-tab.tsx

```diff
@@ -41,12 +47,22 @@ export const DefinitionTab = ({
     );
   }, [entityTypesContext.entityTypes]);
 
+  const onNavigate = (url: string) => {
```

Comment:
Feels like you're replicating a link?

Reply:
See https://github.com/hashintel/hash/pull/2117#discussion_r1115796381